pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1245,7 +1246,8 @@
     ?>  ?=([@ her=ship *] u.parsed)
     =*  her          her.u.parsed
     =/  =peer-state  (got-peer-state her)
-    =/  =channel     [[our her] now channel-state -.peer-state]
+    =/  ping  ?=([[%gall %use %ping *] *] duct)
+    =/  =channel     [[our her] now channel-state ping -.peer-state]
```

Comment:
```suggestion
    =/  ping=?       ?=([[%gall %use %ping *] *] duct)
    =/  =channel     [[our her] now channel-state ping -.peer-state]
```

Here and all other `=/  ping` instances below. Ames has some nice verticals going on, let's keep it up.

Reply:
Also there's a comment saying this is a nasty hack in one place, but not in here and the other places.